_episodes_rmd/04-cond.Rmd

```diff
@@ -274,7 +274,7 @@ In this case, "either" means "either or both", not "either one or the other but
 > `plot_dist` will create a stripchart.
 > A histogram is made with the `hist` command in R.
 >
-> ```{r conditional-challenge-hist, echo=-1}
+> ```{r conditional-challenge-hist, fig.alt = "A grey unlabeled boxplot chart showing the distrubution of inflammation values for day 10 for one inflammation dataset.",  echo=-1}
```

Comment:
I'm adding the alts for the subsequent plots for completion

````suggestion
> ```{r conditional-challenge-hist, fig.alt = c("A grey unlabeled boxplot chart showing the distrubution values between 2 and 9 with a mean at 6.", "A grey unlabeled histogram showing bimodal distribution between 2 and 9 with peaks at 2 and 6.", "A mostly blank strip chart showing five points at 3, 4, 6, 7, and 9"),  echo=-1}
````